Sources_v3/StreamChatUI/ChatChannel/Reactions/ChatMessageReactionVC.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatMessageReactionVC<ExtraData: ExtraDataTypes>: ViewController, UIConfigProvider {
```

Comment:
We use this just for the "big" UI case, right?

Reply:
Yep 👍 The controller comes into the game only when user activates actions on a message (the reactions view is styled as `big`)